ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -240,12 +240,12 @@ else
   CS_NAMESPACE="${OO_INSTALL_NAMESPACE}"
 fi
 
-# The securityContextConfig API field was added in 4.12
+# The securityContextConfig API field was added in 4.12, but the default "enforce" is "restricted" since 4.14
 CS_PODCONFIG=""
 OCP_MINOR_VERSION=$(oc version | grep "Server Version" | cut -d '.' -f2)
-if [ "$OCP_MINOR_VERSION" -gt "11" ]; then
+if [ "$OCP_MINOR_VERSION" -gt "13" ]; then
```

Comment:
If the featuregate TechPreviewNoUpgrade is enable, the catsrc cannot be READY

Reply:
Yes, thanks for the reminder, I will update it later.

Reply:
Add "restricted" since 4.12, thanks!